src/vue/modules/assets/asset-explorer/store/types.spec.js

```diff
@@ -0,0 +1,34 @@
+import { assetExplorerModule } from './index'
+import { types } from './types'
+
+describe('balance explorer module types unit tests', () => {
```

Comment:
overtested. you just test static assignment.

1. firstly you test that all the vuexTypes are present in the module
2. next you test the same thing in the reverse order
3. and the last one you test that the keys are equal to values

1. the only test can pretend being useful, since ensures that you got no unused keys. Having excess vuexType key is a bit not cool, ofcourse, but does not break anything. I am not sure we should slow down our testing and waste additional resources to test a thing that cannot break anything.

2. you ensure that there is no `[undefined] ()`  assignments when test the methods themselves, you should not test it here. this file is only the registry, it should not know anything about the way it is being used, all the usages of invalid registry should not testes by the registry.

3. since all the calls and definitions related to vuexTypes are done only in a way like "vuexTypes.SOME_ACTION" (and should be done in this way only)  you should not test that values and keys are equivalent.